### PR TITLE
update DALEX teaching vignettes

### PR DESCRIPTION
**Pull request by kozaka93**
#43

### CODE REVIEW

File: vignettes/DALEX_teaching2.Rmd [hunk 1]
```diff
@@ -1,176 +1,179 @@
...
+    theme: flatly
+    highlight: kate
+    fig_caption: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message=FALSE, warning=FALSE)
+```
+
+In the vignette below we will see how four selected models of predicting the price of flats find relationships between dependent variables and variables from the model.
```
Comment (jakwisn) [inline vignettes/DALEX_teaching2.Rmd:20]: how a four selected models that predict the price of flats find relationships ...

File: vignettes/DALEX_teaching2.Rmd [hunk 2]
```diff
@@ -1,176 +1,179 @@
...
+```{r libs}
+library(DALEX)
+library(caret)
+library(xgboost)
+library(e1071)
+```
+
+
+#Data set and model
+To illustrate applications of *DALEX* to finding differences between variables we will use an artificial dataset `apartments` available in the DALEX package. Our goal is to predict the price per square meter of an apartment based on selected features such as construction year, surface, floor, number of rooms, district. It should be noted that four of these variables are continuous while the fifth one is a categorical one. Prices are given in Euro.
```
Comment (jakwisn) [inline vignettes/DALEX_teaching2.Rmd:31]: to find differences ....

File: vignettes/DALEX_teaching2.Rmd [hunk 3]
```diff
@@ -1,176 +1,179 @@
...
+sv_svm  <- model_profile(explainer_svm, variable =  "district", type = "partial")
+sv_svm_e1071 <- model_profile(explainer_svm_e1071, variable =  "district", type = "partial")
+sv_tr <- model_profile(explainer_tr, variable =  "district", type = "partial")
+
+
+plot(sv_lm, sv_rf, sv_svm, sv_svm_e1071, sv_tr)
+
+```
+
+For the variable `district` we see that all of the models have nearly the same performance. We see on the plots three clusters: the city center (Srodmiescie), districts well communicated with city center (Ochota, Mokotow, Zoliborz) and other districts closer to city boundaries.
```
Comment (jakwisn) [inline vignettes/DALEX_teaching2.Rmd:174]: We can see three clusters in the plot above:...

File: vignettes/DALEX_teaching.Rmd [hunk 4]
```diff
@@ -138,11 +144,11 @@ For `X5` we want to see `|2 x5 - 1|`.
 All models except the lieanr one are guessing the shape.
```
[the remark this answers is not in the document]
Comment (kozaka93) [reply vignettes/DALEX_teaching.Rmd:144]: Thanks.

File: vignettes/DALEX_teaching2.Rmd (hunk 1 above)
Comment (kozaka93) [reply vignettes/DALEX_teaching2.Rmd:20]: Fixed.

**Pull request merged**